buildconfig/pygame-stubs/image.pyi

```diff
@@ -1,24 +1,18 @@
-from typing import Optional, Tuple, List, Union, IO, Literal
+from typing import IO, Literal, Optional, Sequence, Tuple, Union
 
-from pygame.surface import Surface
 from pygame.bufferproxy import BufferProxy
+from pygame.surface import Surface
 
 _BufferStyle = Union[BufferProxy, bytes, bytearray, memoryview]
-_to_string_format = Literal['p', 'RGB', 'RGBX', 'RGBA', 'ARGB', 'RGBA_PREMULT', 'ARGB_PREMULT']
-_from_buffer_format = Literal['p', 'RGB', 'BRG', 'RGBX', 'RGBA', 'ARGB']
-_from_string_format = Literal['p', 'RGB', 'RGBX', 'RGBA', 'ARGB']
+_to_string_format = Literal["p", "RGB", "RGBX", "RGBA", "ARGB", "RGBA_PREMULT", "ARGB_PREMULT"]
```

Comment:
nitpick: We are moving to use the black python formatter code style which decided on single quotes.

Reply:
I am using the Black formatter as required by mypy, and black converts to double quotes.